### DISCUSSION TITLE
Is there a visual studio implementation?

### DISCUSSION THREAD

**Discussion by blueboy90780** [dotnet/interactive#2084] [Q&A]
I'm using Visual Studio 2022, was looking through the documentation and couldn't find any relevant links for VS specifically. Is there one? If not, does the team plan to implement it into VS?

**Comment by jonsequitur** [comment]
There is an extension  for it in preview: https://marketplace.visualstudio.com/items?itemName=MLNET.notebook

**Reply by blueboy90780** [reply]
Thank you, I've just recently found this extension. Put looking at the source code, is the extension just a pure replica of VSCode extension? There's no specific configurations or settings targeted at Visual Studio IDE. Running this extension on VS 2022 seems much much slower than running on VSCode, even though VS is more performance-heavy, the difference in performance is too large for bottlenecks.
